wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -231,150 +181,223 @@ The number of entries to display for each page. The minimum is **1**, whereas th
+highLevelDescription :: DescriptionEnvironment -> T.Text
+highLevelDescription DescriptionEnvironment{..} = [text|
+This is the specification for the Cardano Wallet API, automatically generated as a [Swagger](https://swagger.io/)
+spec from the [Servant](http://haskell-servant.readthedocs.io/en/stable/) API of [Cardano](https://github.com/input-output-hk/cardano-sl).
 
-instance ToDocs BlockchainHeight where
-  descriptionFor _ = "The height of the blockchain."
+Software Version | Git Revision
+-----------------|-------------------
+$deGitRevision   | $deSoftwareVersion
 
-instance ToDocs SyncProgress where
-  descriptionFor _ = "The sync progress with the blockchain."
+Getting Started
+---------------
 
-instance ToDocs SlotDuration where
-  descriptionFor _ = "The duration for a slot."
+In the following examples, we will use *curl* to illustrate request to an API running on the
+default port **8090**.
 
-instance ToDocs LocalTimeDifference where
-  descriptionFor _ = "The time difference between this node clock and the NTP server."
+> Please note that wallet web API uses TLS for secure communication. Requests to the API need
+to send a client CA certificate that was used when launching the node and identifies the client as
+being permitted to invoke the server API.
 
-instance ToDocs NodeInfo where
-  descriptionFor _ = "A collection of dynamic information for this wallet node."
+Creating a New Wallet
+=====================
 
-instance ToDocs WalletStateSnapshot where
-  descriptionFor _ = "Dump current wallet state"
+You can create your first wallet using the `POST /api/v1/wallets` endpoint as follow:
 
-instance ToDocs (V1 InputSelectionPolicy) where
-  descriptionFor _ = "A policy to be passed to each new `Payment` request to "
-                  <> "determine how a `Transaction` is assembled. "
-                  <> "Possible values: [" <> possibleValuesOf @(V1 InputSelectionPolicy) Proxy <> "]."
+```
+curl -X POST https://localhost:8090/api/v1/wallets                     \
+     -H "Content-Type: application/json; charset=utf-8"                \
+     -H "Accept: application/json; charset=utf-8"                      \
+     --cacert ./scripts/tls-files/ca.crt                               \
+     -d '{                                                             \
+  "operation": "create",                                               \
+  "backupPhrase": ["squirrel", "material", "silly", "twice", "direct", \
+    "slush", "pistol", "razor", "become", "junk", "kingdom", "flee"],  \
+  "assuranceLevel": "normal",                                          \
+  "name": "MyFirstWallet"                                              \
+}'
+```
 
-possibleValuesOf :: (Show a, Enum a, Bounded a) => Proxy a -> T.Text
-possibleValuesOf (Proxy :: Proxy a) = T.intercalate "," . map show $ ([minBound..maxBound] :: [a])
+As a response, the API provides you with a wallet `id` used in subsequent requests to uniquely
```

Comment:
Maybe something to add to subsequent PRs, but would be nice to include:

* Two sentences on the response payload (i.e. JSend, but maybe that's documented later) and specifically about the fact we send automatically pagination information;

* Spend two words on the `assuranceLevel`, even just saying we are going to describe it in more detail later.